plugins/modules/mysql_info.py

```diff
@@ -219,7 +220,8 @@
   description:
     Information about users accounts. The output can be used as an input of the
     mysql_user plugin. Useful when migrating accounts to a new server or to
-    create an inventory. Does not support proxy privileges.
+    create an inventory. Does not support proxy privileges. Cause issue with
+    authentications plugins sha256_password and caching_sha2_password.
```

Comment:
```suggestion
    authentications plugins C(sha256_password) and C(caching_sha2_password).
```
to highlight them a bit